app/controllers/support_interface/countries_controller.rb

```diff
@@ -71,6 +71,8 @@ def country_params
       :teaching_authority_websites_string,
       :teaching_authority_certificate,
       :teaching_authority_other,
+      :teaching_authority_sanction_information,
+      :teaching_authority_status_information,
```

Comment:
Does this need deploying separately first? It sounds like perhaps without it the support interface isn't working correctly?